src/components/InfiniteGrids/InfiniteChannelGrid.tsx

```diff
@@ -20,7 +21,7 @@ type InfiniteChannelGridProps = {
 }
 
 const INITIAL_ROWS = 4
-const INITIAL_CHANNELS_PER_ROW = 4
+const INITIAL_CHANNELS_PER_ROW = 7
```

Comment:
Why this change?

Reply:
@kdembler I wanted to ask about that, why is it a 4, isn't it too few?

Reply:
This is just an initial value, after the first render this should be changed based on the grid's size: https://github.com/Joystream/atlas/blob/personalisation/src/components/InfiniteGrids/InfiniteChannelGrid.tsx#L77

Does changing this value actually impacts the experience? I can imagine because of that maybe we start fetching too few channels at the first render?

Reply:
Yes, I noticed something like that so some placeholder animations would be a bit out of sync, and with a bit more INITIAL_CHANNELS_PER_ROW they sync up better